Documentation/building/testing-with-corefx.md

```diff
@@ -71,36 +71,37 @@ cd ~/corefx/bin/tests/System.Collections.Tests
 ~/coreclr/bin/tests/Linux.x64.Release/testhost/dotnet .\xunit.console.netcore.exe .\System.Collections.Tests.dll -notrait category=nonnetcoretests -notrait category=nonlinuxtests
 ```
 
-### CI Script
-CoreCLR has an alternative way to run CoreFX tests, built for PR CI jobs. 
+## CI Script
+CoreCLR has an alternative way to run CoreFX tests, built for PR CI jobs.
 
 To run tests against pre-built binaries you can execute the following from the CoreCLR repo root:
 
-#### Windows
+### Windows
 1. `.\build.cmd <arch> <build_type> skiptests`
```

Comment:
I realize this was already here, but enumerated lists are automatically computed in markdown. All number can be the same (i.e. all should be 1). This makes adding new ones simple.

Reply:
But it makes reading the raw doc harder. .md files are meant to be pleasant to read in raw form.